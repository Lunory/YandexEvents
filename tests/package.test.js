import packageJson from '../package.json'

describe('package.json', () => {
  it('Проверка снепшота package.json', () => {
    expect(packageJson).toMatchSnapshot()
  })
})
