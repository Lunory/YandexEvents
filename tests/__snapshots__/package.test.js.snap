// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`package.json Проверка снепшота package.json 1`] = `
Object {
  "author": "Yandex.Events Team",
  "cordova": Object {
    "platforms": Array [
      "android",
      "browser",
      "ios",
    ],
    "plugins": Object {
      "cordova-plugin-calendar": Object {
        "CALENDAR_USAGE_DESCRIPTION": "Чтобы вы могли добавлять события к себе в календарь. :)",
      },
      "cordova-plugin-geolocation": Object {},
      "cordova-plugin-whitelist": Object {},
      "yandex-appmetrica-plugin-cordova": Object {},
    },
  },
  "dependencies": Object {
    "axios": "^0.16.2",
    "classnames": "^2.2.5",
    "cordova-android": "~6.2.3",
    "cordova-browser": "^4.1.0",
    "cordova-ios": "~4.4.0",
    "cordova-plugin-calendar": "^4.6.0",
    "cordova-plugin-geolocation": "^2.4.3",
    "cordova-plugin-whitelist": "^1.3.2",
    "history": "^4.7.2",
    "localforage": "^1.5.0",
    "moment": "^2.18.1",
    "react": "^15.6.1",
    "react-dom": "^15.6.1",
    "react-loadable": "^4.0.5",
    "react-redux": "^5.0.6",
    "react-router-dom": "^4.2.2",
    "react-router-page-transition": "^3.0.2",
    "react-slick": "^0.15.4",
    "react-yandex-maps": "^2.1.2",
    "redux": "^3.7.2",
    "redux-thunk": "^2.2.0",
    "styled-components": "^2.1.2",
    "yandex-appmetrica-plugin-cordova": "git+https://github.com/Tamik/metrica-plugin-cordova.git",
    "yandex-ui": "git+https://github.com/Tamik/Yandex-UI.git",
  },
  "description": "Приложение-агрегатор событий для проекта Мобилизация",
  "devDependencies": Object {
    "babel": "^6.23.0",
    "babel-cli": "^6.26.0",
    "babel-core": "^6.26.0",
    "babel-eslint": "^7.2.3",
    "babel-jest": "^20.0.3",
    "babel-loader": "^7.1.2",
    "babel-plugin-module-resolver": "^2.7.1",
    "babel-preset-es2015": "^6.24.1",
    "babel-preset-react": "^6.24.1",
    "babel-preset-stage-2": "^6.24.1",
    "babel-preset-stage-3": "^6.24.1",
    "babel-register": "^6.26.0",
    "css-loader": "^0.28.5",
    "enzyme": "^2.9.1",
    "eslint": "^4.5.0",
    "eslint-config-airbnb": "^15.1.0",
    "eslint-plugin-import": "^2.7.0",
    "eslint-plugin-jest": "^21.0.2",
    "eslint-plugin-jsx-a11y": "^6.0.2",
    "eslint-plugin-react": "^7.3.0",
    "file-loader": "^0.11.2",
    "html-webpack-plugin": "^2.30.1",
    "jest": "^20.0.4",
    "jest-css-modules": "^1.1.0",
    "node-sass": "^4.5.3",
    "prop-types": "^15.5.10",
    "react-addons-test-utils": "^15.6.0",
    "react-hot-loader": "^3.0.0-beta.7",
    "react-test-renderer": "^15.6.1",
    "redux-devtools-extension": "^2.13.2",
    "sass-loader": "^6.0.6",
    "style-loader": "^0.18.2",
    "webpack": "^3.5.5",
    "webpack-babel-jest": "^1.0.4",
    "webpack-dev-server": "^2.7.1",
    "webpack-dotenv-plugin": "^2.0.2",
    "webpack-strip-block": "^0.2.0",
  },
  "displayName": "Яндекс.События",
  "jest": Object {
    "setupFiles": Array [
      "./test-setup.js",
    ],
    "transform": Object {
      ".*": "<rootDir>/node_modules/jest-css-modules",
    },
    "verbose": true,
  },
  "license": "Apache-2.0",
  "main": "index.js",
  "name": "ru.yamblz.events",
  "scripts": Object {
    "android": "cordova run android",
    "android:device": "cordova run android --target device",
    "browser": "cordova run browser --target Chrome",
    "clear": "rm -rf ./www/assets",
    "copy:html": "cp ./src/index.dev.html ./www/index.html",
    "dev": "npm run clear && npm run copy:html && NODE_ENV=development webpack-dev-server --config ./configs/webpack/develop.config.js",
    "ios": "cordova run ios",
    "ios:device": "cordova run ios --target device",
    "lint": "eslint ./src",
    "lint:fix": "eslint --fix",
    "prod": "npm run clear && NODE_ENV=production webpack --config ./configs/webpack/production.config.js",
    "test": "jest",
    "test:update": "jest -u",
  },
  "version": "0.0.1",
}
`;
